.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       with:
        name: mnist-${{ matrix.build_type }}${{ matrix.stats && '-stats' || '' }}
        path: |
-        build/bin/ACC_MNIST*
+        build/bin/ACC*
```

Comment:
Avoid wildcard in artifact path (`build/bin/ACC*`). Export a single `BINARY_PATH` from build and use it explicitly to prevent chmod/run from matching multiple binaries.